config/.env.sample

```diff
@@ -47,6 +47,18 @@ CANVAS_INSTANCE_URL=https://canvas-test.it.umich.edu
 CANVAS_API_CLIENT_ID=
 # Secret for the Canvas Developer Key associated with the application
 CANVAS_API_SECRET=
+# Static Canvas API token for user with admin permissions
+CANVAS_ADMIN_API_TOKEN=
+# Canvas account to which new users will be added (default: 1)
+CANVAS_NEW_USER_ACCOUNT_ID=1
+
+# Invitation API settings
+INVITATION_API_URL=https://apps.cirrusidentity.com/console/api/v1/batchInviteCsv
```

Comment:
I feel this suggestion should be add before this PR get merged, since it is not a whole lot of code change. I feel the configuration file shouldn't have endpoint.

Reply:
I'm sorry I didn't answer you earlier.  I don't disagree with you entirely.  I will make the change.  Yes, I had been considering it.  It has been a very, very low priority, though.  Other things that have been higher priority:

* Updating the response format
* Making sure it works
* Re-updating the response format
* Removing the enrollment processing
* eslint appeasement

All of those things either affect whether my code works or ensures that my code is up to standards maintenance-wise.

This change of whether the URL, or which parts of it, is in the config file doesn't affect those things.

I had originally written the invitation code with the thought that maybe the whole URL should be there in order to make it easier to switch to a service other than Cirrus or change the API function to be used, like a new version, etc.  However, Sam and I had discussed the invitation code a while ago and I made it less general.  I should've moved the URL from the config to the code at that time, but I didn't.  That was an oversight.

Reply:
🏁